app/views/layouts/_flash.html.erb

```diff
@@ -0,0 +1,2 @@
+<p class="notice"><%= notice %></p>
+<p class="alert"><%= alert %></p>
```

Comment:
I told you to name the file _flash_messages. Can you please do that? only flash doesn't make sense. Please give the proper naming to files/variables to increase readability